File: tests/expected/sys_argv.jl

if abspath(PROGRAM_FILE) == @__FILE__
    a = append!([PROGRAM_FILE], ARGS)
    cmd = a[1]
    if cmd == "dart"
        #= pass =#
    else
        @assert(findfirst("sys_argv", cmd) != Nothing)
    end
    if length(a) > 1
        println(a[2])
    else
        println("OK")
    end
end

File: tests/expected/sys_argv.nim
import os
import strutils


proc main() =
  let a: seq[string] = (@[getAppFilename()] & commandLineParams())
  let cmd: string = a[0]
  if cmd == "dart":
    discard
  else:
    assert("sys_argv" in cmd)
  if len(a) > 1:
    echo a[1]
  else:
    echo "OK"

main()
